index.bs

```diff
@@ -322,6 +318,19 @@ it should change the DNS-SD service name to a new value, indicating a new
 mechanism for metadata discovery.
 
 
+Computing the Agent Fingerprint {#computing-agent-fingerprint}
+-------------------------------
```

Comment:
Standardize on section breaks?
```
=======================================================
```
or 
```
-------------------------------
```

Reply:
These for different heading levels (see https://commonmark.org/help/).  If you prefer the # based syntax I guess we could change.